src/status_im2/contexts/quo_preview/avatars/group_avatar.cljs

```diff
@@ -22,7 +22,7 @@
                    (fn [c]
                      {:key   c
                       :value c})
-                   (keys colors/customization))}])
+                   ["#ff0000" "#0000ff"])}])
```

Comment:
Could we avoid the hardcoded string colors here?

Reply:
This is for the preview screen. And it is temporary only.

Reply:
Please add todo

Reply:
@OmarBasem, @Parveshdhull can we start just creating issues in GitHub instead. The issues can just point to where in the code the issue exists 👍

Reply:
There is an issue @J-Son89 on GitHub and linked in the todo